src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -35,30 +35,41 @@ if (DOWNLOAD_ZENODO_TEST_DATA)
                 "a692ba3a6c456d1ad49750717a9d5dda")
 endif()
 
-OPTION(RERUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" OFF)
+OPTION(RERUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" ON)
 find_program(EXE_ISMRMRD_SL NAMES ismrmrd_generate_cartesian_shepp_logan
                 NAMES_PER_DIR
                 PATHS ${CMAKE_INSTALL_PREFIX})
+if (EXE_ISMRMRD_SL STREQUAL "EXE_ISMRMRD_SL-NOTFOUND")
+  set (EXE_ISMRMRD_SL_FOUND OFF)
+else()
+  set (EXE_ISMRMRD_SL_FOUND ON)
+endif()
 
-
-set(TESTDATA_OUTPUT_PATH "${SIRF_SOURCE_DIR}/data/examples/MR/")
-set(TESTDATA_OUTPUT_NAME_FULL "simulated_MR_2D_cartesian.h5")
-set(TESTDATA_OUTPUT_NAME_GRAPPA "simulated_MR_2D_cartesian_Grappa2.h5")
+set(TESTDATA_OUTPUT_PATH "${SHARE_DIR}/data/examples/MR")
```

Comment:
I think this is the output location of the Shepp Logan Phantom data generated during build.
Since we did not want to write them in the data submodule to avoid git problems. 
Does that pose the same write permission problem?

The MR C++ tests write their output directly to wherever the test `exe` is lying.

Reply:
output is in `${SHARE_DIR}`, which is in where everything gets installed. I'd prefer to use `${CMAKE_CURRENT_BINARY_DIR}` really. not sure why that wasn't done. (It also gets rid of all the complications with giving the path to the executable in C++, but still has it for the Python test I guess)